vm_supervisor/conf.py

```diff
@@ -89,39 +89,37 @@ class Settings(BaseSettings):
 
     CONNECTOR_URL = Url("http://localhost:4021")
 
-    CACHE_ROOT = FilePath("/var/cache/aleph/vm")
-    MESSAGE_CACHE = FilePath(join(CACHE_ROOT, "message"))
-    CODE_CACHE = FilePath(join(CACHE_ROOT, "code"))
-    RUNTIME_CACHE = FilePath(join(CACHE_ROOT, "runtime"))
-    DATA_CACHE = FilePath(join(CACHE_ROOT, "data"))
+    CACHE_ROOT = Path("/var/cache/aleph/vm")
+    MESSAGE_CACHE = Path(join(CACHE_ROOT, "message"))
```

Comment:
Could you use the idiomatic way to join paths with pathlib (`Path(dir) / filename`)?